### ISSUE TITLE
[Bug]: Contextual Menu Disabled Check High Contrast icon is not visible

### ISSUE THREAD

**Issue by micahgodbolt** [microsoft/fluentui#22640]
### Library

React / v8 (@fluentui/react)

### Bug Description

Bug introduced in #22518 where checked icon is not visible in high contrast disabled state
